cloudify_helm/tasks.py

```diff
@@ -177,3 +177,43 @@ def inject_env_properties(ctx, **_):
 def update_repo(ctx, **kwargs):
     helm = helm_from_ctx(ctx)
     helm.repo_update(flags=kwargs.get(FLAGS_FIELD))
+
+
+@operation
+@with_helm(ignore_properties_values_file=True)
+def upgrade_release(ctx,
+                    helm,
+                    chart='',
+                    kubeconfig=None,
+                    values_file=None,
+                    set_values=None,
+                    token=None,
+                    flags=None,
+                    **_):
+    """
+    Execute helm upgrade.
+    :param ctx: cloudify context.
+    :param helm: helm client object.
+    :param chart: The chart to upgrade the release with.
+    :param kubeconfig: kubeconfig path.
+    :param values_file: values file path.
+    :return output of `helm upgrade` command
+    """
+    ctx.logger.debug(
+        "Checking if used local packaged chart file, If local file used and "
+        "the command failed check file access permissions.")
+    if os.path.isfile(chart):
+        ctx.logger.info("Local chart file: {path} found.".format(path=chart))
+    output = helm.upgrade(
+        release_name=ctx.node.properties.get(
+            RESOURCE_CONFIG, {}).get(NAME_FIELD),
+        chart=chart,
+        flags=flags,
+        set_values=set_values,
+        values_file=values_file,
+        kubeconfig=kubeconfig,
+        token=token,
+        apiserver=ctx.node.properties.get(
+            CLIENT_CONFIG, {}).get(CONFIGURATION, {}).get(API_OPTIONS, {}).get(
+            HOST), )
+    ctx.instance.runtime_properties['upgrade_output'] = output
```

Comment:
Should we update/override `install_output`? Why or why not?

Reply:
I decided that the output of the upgrade will be string and not json like in the install command,so i put it on different property.
Let me check if there are differences  between the outputs in json format(I need the output will contain `manifest` for future features)